Simplenote/src/main/java/com/automattic/simplenote/TagsActivity.java

```diff
@@ -62,7 +62,6 @@
 
```

Comment:
We can remove the `import com.automattic.simplenote.utils.ThemeUtils;` statement since it is no longer used.